### PR TITLE
Psgo: Allow joinable storage

### PR DESCRIPTION
**Pull request by Volco**

### CODE REVIEW

File: wavelength-plugins/psgo.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+ * Programmed by HoeenHero, and Volco
  */
 
 'use strict';
 
 const CARDS_PER_PACK = 10;
-const cards = require('../config/cards.json');
+let origCards = require('../config/cards.json');
+let newCards = {};
+const fs = require('fs');
```
Comment (DeathlyPlays) [inline wavelength-plugins/psgo.js]: iirc @HoeenHero wants us to use lib/fs

**Comment by DeathlyPlays** [comment]
As long as this is tested I see no other issues 👍

**Comment by Volco** [comment]
it is tested

**Comment by DeathlyPlays** [comment]
Then ya got my 👍

**Pull request merged**